src/clj/witan/send/main.clj

```diff
@@ -5,16 +5,16 @@
             [clojure.string :as string]
             [witan.send.metadata :as md]
             [witan.send.model.output :as so]
-            [witan.send.schemas :as sc]
             [witan.send.send :as send]
             [witan.send.validate-model :as vm]))
 
 (def default-schemas
-  {:schema-inputs {:settings-to-change sc/SettingsToChange
-                   :transitions sc/TransitionCounts
-                   :population sc/PopulationDataset
-                   :costs sc/NeedSettingCost
-                   :valid-states sc/ValidSettingAcademicYears}})
+  {:schema-inputs #_{:settings-to-change sc/SettingsToChange
+                     :transitions sc/TransitionCounts
+                     :population sc/PopulationDataset
+                     :costs sc/NeedSettingCost
+                     :valid-states sc/ValidSettingAcademicYears}
+   {}})
```

Comment:
why the additional empty map?

Reply:
ah don't worry this is the "other bits of old validation hanging around" could you add comments for these please

Reply:
commented